Remove an unneeded declaration of malloc so gcc doesn't complain.
--- a/charresize.c
+++ b/charresize.c
@@ -46,7 +46,6 @@ static int	bit[8] = { 0x80, 0x40, 0x20, 0x10, 0x08, 0x04, 0x02, 0x01 };
 void
 processChar(void)
 {
-  char	*malloc();
   char	*srcimage;
   int	*dstgray;
 
